instrumentation/javalin/javalin-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/javalin/v5_0/JavalinInstrumentation.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.javalin.v5_0;
+
+import static net.bytebuddy.matcher.ElementMatchers.hasSuperType;
```

Comment:
import the same method from `io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers`